ngraph/frontend/onnx_import/src/onnx.cpp

```diff
@@ -44,6 +44,8 @@ namespace ngraph
                 {
                     function->get_output_op(i)->set_friendly_name(
                         graph.get_outputs().at(i).get_name());
+                    function->get_output_op(i)->get_output_tensor(0).set_names(
```

Comment:
@ilyachur do we need to set tensor names for Results operations? I thought output tensors for terminal operations are not suggested to use.

Reply:
Exactly, I think so!

Reply:
OK, I will change that.